help/meson.build:6:6: ERROR: Tried to create target "help-glade-da-update-po", but a target of that name already exists

Index: help/LINGUAS
--- help/LINGUAS.orig
+++ help/LINGUAS
@@ -1,6 +1,5 @@
 # please keep this list sorted alphabetically
 #
-da
 bg
 ca
 cs
